### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by chirimoya**
https://github.com/massiveart/husky/pull/497

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@chirimoya changelog (:

File: src/Sulu/Bundle/AdminBundle/Resources/public/js/components/app/main.js [hunk 1]
```diff
@@ -167,11 +167,18 @@ define(function() {
...
+                    '*actions': 'default'
                 },
 
-                defaultAction: function() {
+                dashboard: function() {
+                    this.html('<div class="sulu-dashboard" data-aura-component="dashboard@suluadmin"/>');
+                }.bind(this),
+
+                default: function() {
                     // We have no matching route
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Resources/public/js/components/app/main.js]: we could also display a error 404 (: message here!

**Comment by wachterjohannes** [comment]
is there no issue for that?

**Comment by chirimoya** [comment]
@wachterjohannes no issue so fare, added changelog entry.

**Pull request merged**